### PR TITLE
[WIP] Docs: 10.2 GenAI what's new updates

### PR DESCRIPTION
**Pull request by nmarrs**
Finalizing whats new content for genAI features for 10.2

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 1]
```diff
@@ -63,9 +63,10 @@ You can now use generative AI to assist you in your Grafana dashboards. So far g
 - **Generate dashboard titles and descriptions** - You can now generate a title and description for your dashboard based on the panels you've added to it. This is useful when you want to quickly create a dashboard and don't want to spend time coming up with a title or description.
 - **Generate dashboard save changes summary** - You can now generate a summary of the changes you've made to a dashboard when you save it. This is useful when you want to quickly save a dashboard and don't want to spend time coming up with a summary.
 
-TODO - how can they use / access these features?? Link to some form of documentation or just say "more info coming soon"?
+To enable these features you will need to first enable the `dashgpt` feature flag in your [Grafana configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles). You then will also need to install and configure Grafana's LLM app plugin. For more information, refer to the [Grafana LLM app plugin documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/llm-plugin/).
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:66]:
```suggestion
To enable these features, you must first enable the `dashgpt` feature flag in your [Grafana configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles). Then install and configure Grafana's LLM app plugin. For more information, refer to the [Grafana LLM app plugin documentation](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/llm-plugin/).
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 2]
```diff
@@ -63,9 +63,10 @@ You can now use generative AI to assist you in your Grafana dashboards. So far g
...
 
-TODO: Add image / gif / video
+TODO: Add image / gif / video (enablement?)
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:68]:
```suggestion
```

**Review by imatwawana — APPROVED** [review]
Couple wording things and I'm pretty sure there was a TODO left in that was meant to be removed. Otherwise, looks good!

**Comment by nmarrs** [comment]
Address by [this commit on the main whats new branch](https://github.com/grafana/grafana/pull/75909/commits/f1f27f66050aacfbb5aafa62b757ba84d4e62d12)

**Pull request closed without merging**